src/services/apes/apes.ts

```diff
@@ -1,15 +1,20 @@
 import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
-import { EndowmentBalances, Token } from "types/aws";
+import { PaymentIntent } from "@stripe/stripe-js";
+import { EndowmentBalances, FiatCurrencyData, KYCData, Token } from "types/aws";
 import { ChainID } from "types/chain";
-import { appRoutes } from "constants/routes";
+import { version as v } from "services/helpers";
```

Comment:
nit: as apes is also inside `/services`
```suggestion
import { version as v } from "../helpers";
```